frontend/src/components/Playback/UpNext.vue

```diff
@@ -13,13 +13,12 @@
         xl="3">
         <v-card class="pointer-events-all">
           <v-card-title class="text-h6">
-            <i18n path="dialog.upNext.nextItemPlayingIn" tag="span">
-              <template #time>
-                <span class="text-primary darken-2">
-                  {{ $tc('units.time.seconds', currentItemTimeLeft) }}
-                </span>
-              </template>
-            </i18n>
+            <span>
+              {{ $t('dialog.upNext.nextItemPlayingIn') }}
+              <span class="text-primary darken-2">
+                {{ $tc('units.time.seconds', currentItemTimeLeft) }}
+              </span>
+            </span>
```

Comment:
This is not strictly the original code, but as this component is still not plugged-in to the video player, it will probably need a refactor anyway.

The time should be passed as a prop as well, it looks strange to me to pass this as an slot.